### DISCUSSION TITLE
Wiki Documentation 1.2

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#118] [Ideas]
To contributor v1.2 : @SyaefulKai 

Untuk feat yang sudah dibuat, bisa dibuatkan contoh example penggunaannya kah untuk saya tambahkan ke Wiki?

Terimakasih

**Comment by SyaefulKai** [comment]
Bisa pak

**Comment by SyaefulKai** [comment]
Untuk POST & PUT Patientnya belum ditambahkan pak.

Untuk update wikinya bisa dilihat [di sini](https://github.com/SyaefulKai/satusehat-integration/blob/wiki-update/Usage.md)

Kalau ada kekurangan/kesalahan silahkan berkomentar.

**Comment by ivanwilliammd** [comment]
Untuk FHIR SATUSEHAT pakai yg R4 bukan R5... boleh coba dikoreksi kembali

**Comment by SyaefulKai** [comment]
Sudah dikoreksi pak.

**Comment by ivanwilliammd** [comment]
Sip mantap

**Comment by ivanwilliammd** [comment]
In case ada tambahan lainnya feel free untuk info di issue thread ini, nanti akan sy sync ke wiki

**Comment by SyaefulKai** [comment]
Maaf pak, itu ada kesalahan di Condition. Condition tidak ada method setPerformer.

**Comment by ivanwilliammd** [comment]
Oke updated
